helpers/escape-elastic-search.js

```diff
@@ -2,7 +2,7 @@
 
 function escapeElasticSearch(query) {
   return query
-    .replace(/[\*\+\-"~\?^{}\(\):!\/[\]\\]/g, '\\$&')
+    .replace(/[*+\-"~?^{}():!/[\]\\]/g, '\\$&')
```

Comment:
This example is a bit tricky. See:

```
$ node
> 'yes\ no'
'yes no'
> 'yes\\ no'
'yes\\ no'
> 'yes\ no' === 'yes no'
true
```

In other words, the `\` in `'yes\ no'` is escaping the following whitespace. As `'\ '` has no meaning in JavaScript, `'\ ' === ' '` is true.